client/js/widgets/button.js

```diff
@@ -53,12 +53,19 @@ export class Button extends Widget {
 
     if(this.p('debug'))
       $('#debugButtonOutput').textContent = '';
-
+   let thisID = this.p('id');
```

Comment:
The indentation is missing a space.

This can also be `const` because it will never change.